catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/transformer/transformers/MiroTransformableTransformer.scala

```diff
@@ -215,46 +217,6 @@ class MiroTransformableTransformer
     miroIDList ++ sierraList ++ libraryRefsList
   }
 
-  /* Populate the subjects field.  This is based on two fields in the XML,
-   *  <image_keywords> and <image_keywords_unauth>.  Both of these were
-   *  defined in part or whole by the human cataloguers, and in general do
-   *  not correspond to a controlled vocabulary.  (The latter was imported
-   *  directly from PhotoSoft.)
-   *
-   *  In some cases, these actually do correspond to controlled vocabs,
-   *  e.g. where keywords were pulled directly from Sierra -- but we don't
-   *  have enough information in Miro to determine which ones those are.
-   */
-  private def getSubjects(miroData: MiroTransformableData): List[Subject] = {
-    val keywords: List[Subject] = miroData.keywords match {
-      case Some(k) =>
-        k.map { keyword =>
-          Subject(label = keyword, concepts = List(Concept(keyword)))
-        }
-      case None =>
-        List()
-    }
-
-    val keywordsUnauth: List[Subject] = miroData.keywordsUnauth match {
-      case Some(k) =>
-        k.map { keyword =>
-          Subject(label = keyword, concepts = List(Concept(keyword)))
-        }
-      case None =>
-        List()
-    }
-
-    keywords ++ keywordsUnauth
-  }
-
-  private def getGenres(miroData: MiroTransformableData): List[Genre] = {
-    // Populate the subjects field.  This is based on two fields in the XML,
-    // <image_phys_format> and <image_lc_genre>.
-    (miroData.physFormat.toList ++ miroData.lcGenre.toList).map { g =>
-      Genre(label = g, concepts = List(Concept(g)))
-    }.distinct
-  }
-
```

Comment:
Breaking up the MiroTransformableTransformer has been an open TODO for a while, so I snuck in a change to break out Genres and Subjects while I was here.